CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Next
 
+## 3.14.1 - 2025-04-21
```

Comment:
do you need to bump the version anywhere too or does CI handle it?

Reply:
https://github.com/PostHog/posthog-android/blob/main/RELEASING.md